x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -476,7 +487,7 @@ export class TaskRunner<
       triggeredActionsStatus: ActionsCompletion.COMPLETE,
     };
 
-    if (!muteAll && this.shouldLogAndScheduleActionsForAlerts()) {
+    if (!this.isRuleSnoozed(rule) && this.shouldLogAndScheduleActionsForAlerts()) {
```

Comment:
nit: Should we calculate 
```
const isRuleSnoozed = this.isRuleSnoozed(rule);
```

and use that instead of calling it twice? I notice `isRuleSnoozed` uses `Date.now()` so there is a tiny tiny chance that these two calls could return different results :)

Reply:
ya, sounds right, thx!

Reply:
fixed in commit d865d70819de170613acfa8fb319dd3039f03fe3